src/main/java/seedu/duke/entities/Order.java

```diff
@@ -3,7 +3,7 @@
 import java.util.List;
 
 public class Order {
-    private List<Dish> dishes;
+    private static List<Dish> dishes;
```

Comment:
why static? if static, all your orders will have the same dishes. 
revert.

Reply:
OK